benchmarks/distributed/rpc/parameter_server/launcher.py

```diff
@@ -19,9 +19,9 @@
 USE_CUDA_RPC = "use_cuda_rpc"
 
 
-def get_name(rank, configs):
-    t_count = configs.trainer_count
-    ps_count = configs.ps_count
+def get_name(rank, config):
```

Comment:
nit: we usually focus on a specific thing in individual PRs. For example, if you would like to modify arg names to simplicity or better readability, those modifications themself can be a separate PR. I am just flagging this. No need to split that from the current PR.

Reply:
okay.